samples/Gallery/Shared/Samples/TextShapingSample.cs

```diff
@@ -1,4 +1,5 @@
-﻿using SkiaSharp;
+﻿#if !HAS_UNO
```

Comment:
Many of these samples were disabled because of missing mono support, but it should now work properly.

Reply:
Yeah, also saw some were using webp, which was originally excluded from the canvaskit builds. Added that back in and now we can load those too.